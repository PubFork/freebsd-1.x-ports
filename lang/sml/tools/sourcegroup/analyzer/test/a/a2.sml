signature A = sig end

(* dsi A *)
