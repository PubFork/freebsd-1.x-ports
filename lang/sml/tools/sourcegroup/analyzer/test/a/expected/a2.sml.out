source sml "a/a2.sml"
  export signature A ;

